autometa/config/databases.py

```diff
@@ -66,230 +97,504 @@ class Databases:
             'archaea_single_copy_cutoffs',
         ],
     }
-
-    def __init__(self, config=DEFAULT_CONFIG, dryrun=False, nproc=mp.cpu_count()):
-        if type(config) is not ConfigParser:
+    def __init__(self, config=DEFAULT_CONFIG, dryrun=False, nproc=mp.cpu_count(),
+        do_update=False):
+        if not isinstance(config, ConfigParser):
             raise TypeError(f'config is not ConfigParser : {type(config)}')
-        if type(dryrun) is not bool:
-            raise TypeError(
-                f'dryrun must be True or False. type: {type(dryrun)}')
-
-        self.config = config
-        self.dryrun = dryrun
-        self.nproc = nproc
-        self.prepare_sections()
+        if not isinstance(dryrun, bool) and isins:
```

Comment:
I couldn't find `isins` in the Python documentation. Was this supposed to be an `isinstance` clause?

Reply:
This is an oversight. 🐛